### ISSUE TITLE
Getting negative loss for keras "reuters" dataset

### ISSUE THREAD

**Issue by KaushikNathMIT** [ilivans/tf-rnn-attention#10]
I am trying out the code with imdb reuters example, which has 46 classes. I am getting highly negative loss on each iteration.

**Comment by ilivans** [comment]
As mentioned in the `train.py` docstring the model is devised for **binary** classification (output shape, loss function). So specify please what changes you made first to apply the model to multi-label (or multi-class) problem.
